ds-caselaw-ingester/lambda_function.py

```diff
@@ -439,15 +435,17 @@ def handler(event, context):
                 tar,
                 f"{consignment_reference}/{image_filename}",
                 image_filename,
-                uri,
+                target_uri,
                 s3_client,
             )
 
     # Copy original tarfile
-    store_file(open(filename, mode="rb"), uri, os.path.basename(filename), s3_client)
+    store_file(
+        open(filename, mode="rb"), target_uri, os.path.basename(filename), s3_client
+    )
 
-    if api_client.get_published(uri):
-        update_published_documents(uri, s3_client)
+    if api_client.get_published(target_uri):
```

Comment:
Can we get rid of this block now that we should never be updating documents anymore, just like you removed the marklogic update calls above?

Reply:
It took me a while to understand that we don't ever need to worry about the target URI being published, so this is  indeed all rubbish. 🔥